src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain
+
+class RacingGame(
+    private var cars: Cars,
+    private val tryCount: TryCount,
+) {
+    private var recordCars: MutableList<Cars> = mutableListOf()
```

Comment:
```suggestion    
    private val recordCars: MutableList<Cars> = mutableListOf()
```
val로 작성해도 될 프로퍼티로 보입니다.

참고로 List 타입을 사용하실 때에는 var로 새로운 인스턴스를 만들어 값을 할당하는 것보다 Mutable 리스트를 조작하는 것이 비용이 더 저렴할때가 많습니다.

Reply:
recordCars라는 네이밍이 '차를 기록하다'라는 의미로 받아들여질 수도 있을 것 같은데요, carRecords와 같은 네이밍은 어떨까요?